mod-source-record-manager-server/src/main/resources/rules/marc_holding_rules.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
It's not critical for back-end, but Khalilah mentioned, that we use mark holdings for the singular. So, in this case we should rename to `marc_holdings_rules.json`